src/lib/components/Island.svelte

```diff
@@ -20,15 +20,15 @@
 
 <style lang="scss">
   @use "../styles/mixins/media";
+  @use "../styles/mixins/content";
 
   .island {
-    background: var(--focus-background);
-    color: var(--focus-background-contrast);
+    @include content.island;
 
     height: 100%;
 
     ::-webkit-scrollbar {
-      background: var(--focus-background);
+      @include content.island;
```

Comment:
Why is this needed?

Reply:
Because the component and the utility share the same colors. As we don't have component related style in the dark/light/variable scss, I used a mixin to avoid duplication so that if one day we change the island colors, it changes the color in both.

Reply:
But why is it also needed inside `::-webkit-scrollbar`? It's already above

Reply:
Does it style the scrollbar in the same way of the container and that's needed?

Reply:
It styles the background of the scrollbar. The "color" field of the mixin is not use but including it spare duplicating the background property

<img width="1536" alt="Capture d’écran 2022-11-29 à 12 58 20" src="https://user-images.githubusercontent.com/16886711/204523398-6b7c2e6e-40b0-4d53-879e-9e819fb0325e.png">

Reply:
ok, thanks for the clarification